src/help/zaphelp/contents/cmdline.html

```diff
@@ -9,6 +9,14 @@
 <BODY BGCOLOR="#ffffff">
 <H1>Command Line</H1>
 
+To run ZAP via the command line, you will need to locate the `zap.sh` script. On Mac, this will be at:<br/>
+<pre>/Applications/OWASP\ ZAP.app/Contents/Java/zap.sh</pre>
+On Linux, it will be at<br/>
+<pre>/full/path/to/ZAP/zap.sh</pre>

```

Comment:
I'd be tempted to have a separate page for details of where ZAP is installed as we can then link to it from other pages.
Windows should definitely be included as the first example as thats the platform the majority of our users use.
On recent Windows its in "C:\Program Files (x86)\OWASP\Zed Attack Proxy"
We could just say that on Linux and any OS where the 'cross platform' or weekly release has been installed then it will be relate to the place they installed it